test/entropic/sinkhorn_stabilized.jl

```diff
@@ -0,0 +1,100 @@
+using OptimalTransport
+
+using Distances
+using ForwardDiff
+using LogExpFunctions
+using PythonOT: PythonOT
+
+using LinearAlgebra
+using Random
+using Test
+
+const POT = PythonOT
+
+Random.seed!(100)
+
+@testset "sinkhorn_stabilized.jl" begin
+    # size of source and target
+    M = 250
+    N = 200
+
+    # create two random histograms
+    μ = normalize!(rand(M), 1)
+    ν = normalize!(rand(N), 1)
+
+    # create random cost matrix
+    C = pairwise(SqEuclidean(), rand(1, M), rand(1, N); dims=2)
```

Comment:
The latter does not work since the `pairwise` implementation in Distances requires a matrix, not a vector. Of course, one could use `rand(M, 1)` etc., I'm just more used to `dims=2` since it was the default (and still is even though default values are deprecated).